src/frontend/src/pages/HomePage/components/GuestOrganizationInfo.tsx

```diff
@@ -0,0 +1,57 @@
+import { Box, Typography } from '@mui/material';
+import { Grid } from '@mui/material';
+import { useCurrentOrganization } from '../../../hooks/organizations.hooks';
+import React from 'react';
+import { NERButton } from '../../../components/NERButton';
+import InfoRoundedIcon from '@mui/icons-material/InfoRounded';
+
+interface GuestOrganizationInfoButtonProps {
+  href?: string;
+  buttonText: string;
+}
+
+const NERGuestButton: React.FC<GuestOrganizationInfoButtonProps> = ({ href, buttonText }) => {
+  return (
+    <Grid item xs={4}>
+      <NERButton variant="contained" fullWidth={true} style={{ justifyContent: 'flex-start' }} href={href}>
+        <InfoRoundedIcon style={{ color: 'white' }} />
+        <Typography color="white" sx={{ marginLeft: 1 }}>
+          {buttonText}
+        </Typography>
+      </NERButton>
+    </Grid>
+  );
+};
+
+const GuestOrganizationInfo = () => {
+  const organizationName = useCurrentOrganization().data?.name;
```

Comment:
you want to call the hook at the took of the component like this:
{data: organization, isLoading, isError, error} = useCurrentOrganization();

and then you want to check if is loading or is error is true. Take a look at how we set this up in some of our other frontend components with hooks. We should only have to call the hook once, and we want to ensure data is defined and not 'data?'.